packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionHistorySearchResults.tsx

```diff
@@ -34,19 +34,24 @@ function useTransactionHistoryUpdater() {
     runFetcher: true
   })
 
-  const { transactions, updatePendingTransaction } = transactionHistoryProps
+  const { transactions, loading, updatePendingTransaction } =
+    transactionHistoryProps
 
   const pendingTransactions = useMemo(() => {
     return transactions.filter(isTxPending)
   }, [transactions])
 
   useEffect(() => {
     const interval = setInterval(() => {
-      pendingTransactions.forEach(updatePendingTransaction)
+      // only update pending transactions when tx history is not loading
```

Comment:
I think we can remove this update from the current PR to keep diffs relevant.
Let's open a separate PR for these.